packages/react-router/lib/dom/ssr/links.ts

```diff
@@ -421,27 +421,6 @@ export function getNewMatchesForLinks(
 }
 
 export function getModuleLinkHrefs(
-  matches: AgnosticDataRouteMatch[],
-  manifestPatch: AssetsManifest
-): string[] {
-  return dedupeHrefs(
-    matches
-      .map((match) => {
-        let route = manifestPatch.routes[match.route.id];
-        let hrefs = [route.module];
-        if (route.imports) {
-          hrefs = hrefs.concat(route.imports);
-        }
-        return hrefs;
-      })
-      .flat(1)
-  );
-}
-
-// The `<Script>` will render rel=modulepreload for the current page, we don't
-// need to include them in a page prefetch, this gives us the list to remove
-// while deduping.
-function getCurrentPageModulePreloadHrefs(
```

Comment:
Apart from the manifest arg having a different name, this function was identical to `getModuleLinkHrefs` above. I went with `manifest` rather than `manifestPatch` as the canonical arg name.